README.md

```diff
@@ -7,73 +7,143 @@
 [codecov-url]:https://codecov.io/gh/moxystudio/next-sitemaps-plugin
 [codecov-image]:https://img.shields.io/codecov/c/github/moxystudio/next-sitemaps-plugin/master.svg
 
-This plugin enables your next.js application to have working sitemaps so that search engine crawlers can index all your pages.
+This module provides an API handler and a plugin that enables your Next.js application to have working sitemaps so that search engine crawlers can index all your pages.
+
+## Installation
+
+```cmd
+npm install @moxy/next-sitemaps
+```
 
 ## Usage
 
 ### Simple usage
 
+**1. Add the plugin to `next.config.js`**
+
 ```js
+const withSitemap = require('@moxy/next-sitemaps/plugin');
+
 withPlugins([
     //other plugins..
-    withSitemaps()
+    withSitemap()
     //other plugins...
-])
+]);
 ```
 
-You can also specify options to this plugin:
+**2. Add the API handler on `pages/api/sitemap.js`**
 
-| Parameter         | Default                | Links                |
-|:-------------|:--------------------------:|--------------------------:|
-| baseUrl           | `/`                   | [More info](#specifying-the-base-url)
-| sitemapsLocation  | `public/sitemaps.xml` | [More info](#specifying-the-target-location)
-| mapDynamicRoutes  | `null`                | [More info](#handling-dynamic-routing)
+```js
+import createSitemapApiHandler from '@moxy/next-sitemaps';
 
-### Specifying the base url
+export default createSitemapApiHandler({
+    baseUrl: 'https://moxy.studio',
+    mapDynamicRoutes: {
+        '/[id]': () => ['id1', 'id2'],
+    },
+});
+```
 
-```js
-withSitemaps({
-    baseUrl: 'https://moxy.studio'
-}),
+**3. Add the endpoint URL to your project's robots.txt file**
+
+```txt
+Sitemap: https://moxy.studio/api/sitemap
+User-agent:*
+Disallow:
+
+...
```

Comment:
```suggestion
# ...
```

Reply:
Why?

Reply:
Because that’s a comment. All examples mentioning that you may have adiciontal things, is through a comment like // ...

Reply:
Got it, to keep it valid 👍